db/migrate/20220707152130_create_cars.rb

```diff
@@ -0,0 +1,14 @@
+class CreateCars < ActiveRecord::Migration[7.0]
+  def change
+    create_table :cars do |t|
+      t.string :name
+      t.text :description
+      t.string :image_url
+      t.string :city
+      t.integer :user_id
```

Comment:
- [ ]  Also, I see a user_id in the _create_cars table. What was the idea? Are you trying to create a user_id as a foreign key? Or could it be a typo instead of car_id? Kindly check and rectify that.